components/aws/sagemaker/deploy/README.md

```diff
@@ -0,0 +1,62 @@
+# SageMaker Hosting Services - Create Endpoint Kubeflow Pipeline component
+## Summary
+Component to deploy a model in Sagemaker Hosting Service from a Kubeflow Pipelines workflow.
+
+# Details
+Deploying a model using Amazon SageMaker hosting services is a three-step process:
+
+1. **Create a model in Amazon SageMaker** - Specify the S3 path where model artifacts are stored and Docker registry path for the image that contains the inference code 
+2. **Create an endpoint configuration for an HTTPS endpoint** - Specify the name of model in production variants and the type of instance that you want Amazon SageMaker to launch to host the model.
+3. **Create an HTTPS endpoint** - Launch the ML compute instances and deploy the model as specified in the endpoint configuration
+
+This component handles Step 2 and 3.
```

Comment:
Lets point to the docs for Step 1.

Reply:
Agreed. If i just open this readme, this sentence is unclear to me.

Reply:
done